pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -35,14 +36,17 @@ const (
 	// DefaultBindTimeoutSeconds defines the default bind timeout in seconds
 	DefaultBindTimeoutSeconds = 600
 
-	allBoundStateKey framework.StateKey = "volumebinding:all-bound"
+	stateKey framework.StateKey = Name
```

Comment:
Unrelated to this PR, but could we potentially use this state mechanism to get rid of the PodBindingCache, which is only relevant for a single scheduling cycle?

Reply:
Yes, this is planned https://github.com/kubernetes/kubernetes/issues/90962